packages/ui-extensions/buildTargetDts.ts

```diff
@@ -234,7 +244,17 @@ export function buildTargetsDefinitions(directory: string, surface: string) {
   const project = new Project();
   const buildPath = resolve(directory, 'build');
   const srcPath = resolve(directory, `src/surfaces/${surface}`);
-  const success = copyComponentDefinitions({srcPath, buildPath, surface});
+
+  const checkoutSrcPath = resolve(directory, `src/surfaces/checkout`);
+
+  const componentSrcPaths =
```

Comment:
I would prefer that we pass things in from loom instead of hardcoding custom stuff here.
```suggestion
  const componentSrcPaths = [
    srcPath,
    ...additionalComponentPaths.map((path) => resolve(directory, path)),
  ];
```

And then in the loom config you can do
```ts
buildTargetsDefinitions(
  resolve(process.cwd(), 'packages/ui-extensions'),
  'customer-account',
  ['src/surfaces/checkout'],
);

Reply:
ah! thanks for the suggestion, will make a update 🙇

Reply:
updated!